Calling try
In try
Execution completed
